sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/active/directory/implementation/autoconfigure/b2c/AADB2CProperties.java

```diff
@@ -157,6 +153,33 @@ private void validateCommonProperties() {
         }
     }
 
+    /**
+     * Validate URL properties configuration.
+     */
+    private void validateURLProperties() {
+        if (!CollectionUtils.isEmpty(userFlows)) {
+            if (!StringUtils.hasText(tenant) && !StringUtils.hasText(baseUri)) {
```

Comment:
`!StringUtils.hasText(tenant)` is not belong to `validateURLProperties`. The method name will make the code confusing.

Reply:
updated